npm-packages/eslint-config-meteor/index.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
     'no-underscore-dangle': [
       'error',
       {
-        allow: ['_id', '_ensureIndex'],
+        allow: ['_id', 'ensureIndexAsync'],
```

Comment:
`ensureIndexAsync` doesn't need to be listed since it doesn't start with an underscore.

Reply:
This actually is a breaking change for eslint-config-meteor. We should leave _ensureIndex here since it would still be wanted for any apps on older versions of Meteor.